src/components/InstructorStatsWidget/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import {map} from 'lodash'
+import {statsTitleText} from 'utils/text'
+import Widget from 'components/Widget'
+import List from 'components/List'
+import IconLabel from 'components/IconLabel'
+
+export default ({publishedLessons, publishedCourses}) => {
+
+  if(!publishedLessons) {
+    return null
+  }
+
+  const items = [
+    {
+      type: 'lesson',
+      text: `${publishedLessons} published lessons`,
+    },
+    {
+      type: 'course',
+      text: `${publishedCourses} published courses`,
+    },
+  ]
+
+  return (
+    <Widget title={statsTitleText}>
+      <List items={map(items, (item, index) => (
+        <IconLabel
+          iconType={item.type}
+          labelText={item.text}
+        />
+      ))} />
+    </Widget>
```

Comment:
Most `screen/*/components` files are widgets like this now; they are self-contained with all the sub-components, utilities and `WrappedRequests` when needed so they can be moved or promoted simply by moving the directory.